### PR TITLE
fix: small typo in remote update docs example

### PR DESCRIPTION
**Pull request by lovelyunsh**
Closes #

_Describe your proposed changes here._
typo on https://docs.influxdata.com/influxdb/v2.7/reference/cli/influx/remote/update/#Copyright
`remote remote` -> `remote update`

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

**Comment by lovelyunsh** [comment]
!signed-cla

**Comment by lovelyunsh** [comment]
!signed-cla

File: content/influxdb/v2.7/reference/cli/influx/remote/update.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ influx remote update [command options] [arguments...]
    ```
 2. Use the following command to update the remote:
     ```sh
-    influx remote remote \
+    influx remote update \
       --id 0ooxX0xxXo0x
       --name new-example-name
       --description new-examle-description
```
Comment (jstirnaman) [inline content/influxdb/v2.7/reference/cli/influx/remote/update.md:61]:
```suggestion
      --description new-example-description
```

File: content/influxdb/v2.6/reference/cli/influx/remote/update.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ influx remote update [command options] [arguments...]
    ```
 2. Use the following command to update the remote:
     ```sh
-    influx remote remote \
+    influx remote update \
       --id 0ooxX0xxXo0x
       --name new-example-name
       --description new-examle-description
```
Comment (jstirnaman) [inline content/influxdb/v2.6/reference/cli/influx/remote/update.md:61]:
```suggestion
      --description new-example-description
```

File: content/influxdb/v2.5/reference/cli/influx/remote/update.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ influx remote update [command options] [arguments...]
    ```
 2. Use the following command to update the remote:
     ```sh
-    influx remote remote \
+    influx remote update \
       --id 0ooxX0xxXo0x
       --name new-example-name
       --description new-examle-description
```
Comment (jstirnaman) [inline content/influxdb/v2.5/reference/cli/influx/remote/update.md:61]:
```suggestion
      --description new-example-description
```

File: content/influxdb/v2.4/reference/cli/influx/remote/update.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ influx remote update [command options] [arguments...]
    ```
 2. Use the following command to update the remote:
     ```sh
-    influx remote remote \
+    influx remote update \
       --id 0ooxX0xxXo0x
       --name new-example-name
       --description new-examle-description
```
Comment (jstirnaman) [inline content/influxdb/v2.4/reference/cli/influx/remote/update.md:61]:
```suggestion
      --description new-example-description
```

File: content/influxdb/v2.3/reference/cli/influx/remote/update.md [hunk 5]
```diff
@@ -55,7 +55,7 @@ influx remote update [command options] [arguments...]
    ```
 2. Use the following command to update the remote:
     ```sh
-    influx remote remote \
+    influx remote update \
       --id 0ooxX0xxXo0x
       --name new-example-name
       --description new-examle-description
```
Comment (jstirnaman) [inline content/influxdb/v2.3/reference/cli/influx/remote/update.md:61]:
```suggestion
      --description new-example-description
```

File: content/influxdb/v2.2/reference/cli/influx/remote/update.md [hunk 6]
```diff
@@ -55,7 +55,7 @@ influx remote update [command options] [arguments...]
    ```
 2. Use the following command to update the remote:
     ```sh
-    influx remote remote \
+    influx remote update \
       --id 0ooxX0xxXo0x
       --name new-example-name
       --description new-examle-description
```
Comment (jstirnaman) [inline content/influxdb/v2.2/reference/cli/influx/remote/update.md:61]:
```suggestion
      --description new-example-description
```

**Review by jstirnaman — APPROVED** [review]
@lovelyunsh Thanks for submitting this fix. I added a few more.

**Pull request merged**